core/server/controllers/admin.js

```diff
@@ -32,27 +30,7 @@ adminControllers = {
             });
         }
 
-        updateCheck().then(function then() {
-            return updateCheck.showUpdateNotification();
-        }).then(function then(updateVersion) {
-            if (!updateVersion) {
-                return;
-            }
-
-            var notification = {
-                type: 'upgrade',
-                location: 'settings-about-upgrade',
-                dismissible: false,
-                status: 'alert',
-                message: i18n.t('notices.controllers.newVersionAvailable',
-                                {version: updateVersion, link: '<a href="http://support.ghost.org/how-to-upgrade/" target="_blank">Click here</a>'})};
-
-            return api.notifications.browse({context: {internal: true}}).then(function then(results) {
-                if (!_.some(results.notifications, {message: notification.message})) {
-                    return api.notifications.add({notifications: [notification]}, {context: {internal: true}});
-                }
-            });
-        }).finally(function noMatterWhat() {
+        updateCheck().finally(function noMatterWhat() {
```

Comment:
Agreed, this doesn't seem to be something for which a normal request needs to be delayed? IMO this is something that would be better off as a background job so that the update check service being down doesn't have a noticeable affect on every Ghost install. We also have a smart client so having the notification available immediately is not so much of a concern, the client can request in the background once the admin interface has loaded so there's no chance of a 10+ sec delay when trying to access the admin screen.